diff --git a/g10/decrypt-data.c b/g10/decrypt-data.c
index 4bb2b7e..5d250bf 100644
--- a/g10/decrypt-data.c
+++ b/g10/decrypt-data.c
@@ -282,7 +282,7 @@ decrypt_data (ctrl_t ctrl, void *procctx, PKT_encrypted *ed, DEK *dek,
     }
 
   write_status_printf (STATUS_DECRYPTION_INFO, "%d %d %d",
-                       ed->mdc_method, dek->algo, 0);
+                       ed->mdc_method, dek->algo, ed->aead_algo);
 
   if (opt.show_session_key)
     {
